stdlib/stdlib.mli

```diff
@@ -298,6 +298,11 @@ external __POS_OF__ : 'a -> (string * int * int * int) * 'a = "%loc_POS"
     @since 4.02.0
  *)
 
+external __SCOPE__ : string = "%loc_SCOPE"
+(** [__SCOPE__] returns the name of the current scope, where a scope corresponds
+    to named functions, anonymous functions, modules, value bindings, classes,
+    methods, etc. *)
```

Comment:
Needs a `@since` annotation.

I don't think this makes clear that `__SCOPE__` includes the name of the surrounding module. Here's what I wrote on my branch, feel free to mix and match:
```
[__SCOPE__] returns the name of the current function or method,
including any enclosing modules or classes.
```

Reply:
Thanks, I added the annotation and stole the better docstring from your branch.